internal/ethereum/send_transaction.go

```diff
@@ -72,13 +72,16 @@ func (c *ethConnector) mapGasPrice(ctx context.Context, input *fftypes.JSONAny,
 		return nil
 	}
 	gasPriceObject := input.JSONObjectNowarn()
-	tx.MaxPriorityFeePerGas = (*ethtypes.HexInteger)(gasPriceObject.GetInteger("maxPriorityFeePerGas"))
-	tx.MaxFeePerGas = (*ethtypes.HexInteger)(gasPriceObject.GetInteger("maxFeePerGas"))
+	maxPriorityFeePerGas := (*ethtypes.HexInteger)(gasPriceObject.GetInteger("maxPriorityFeePerGas"))
+	maxFeePerGas := (*ethtypes.HexInteger)(gasPriceObject.GetInteger("maxFeePerGas"))
 	if tx.MaxPriorityFeePerGas.BigInt().Sign() > 0 || tx.MaxFeePerGas.BigInt().Sign() > 0 {
```

Comment:
This logic seems to be broken, now - should this be...

```suggestion
	if maxPriorityFeePerGas.BigInt().Sign() > 0 || maxFeePerGas.BigInt().Sign() > 0 {
```